### ISSUE TITLE
Namespace support

### ISSUE THREAD

**Issue by alexey-pelykh** [lukeautry/tsoa#74]
As it turns out, following won't work:
```
export interface TestModel extends Model {
//...
  modelsObjectIndirectNS: TestSubModelContainerNamespace.TestSubModelContainer;
  modelsObjectIndirectNS2: TestSubModelContainerNamespace.TestSubModelContainer2;
  modelsObjectIndirectNS_Alias: TestSubModelContainerNamespace_TestSubModelContainer;
  modelsObjectIndirectNS2_Alias: TestSubModelContainerNamespace_TestSubModelContainer2;
//...
}

export namespace TestSubModelContainerNamespace {
  export interface TestSubModelContainer {
    [key: string]: TestSubModel2;
  }

  export interface TestSubModelContainer2 {
    [key: string]: TestSubModel2;
  }
}
export type TestSubModelContainerNamespace_TestSubModelContainer = TestSubModelContainerNamespace.TestSubModelContainer;
export type TestSubModelContainerNamespace_TestSubModelContainer2 = TestSubModelContainerNamespace.TestSubModelContainer2;
```

Hopefully, I'll be able to address that with PR, yet if anyone willing to give a hint, that would be great

**Comment by alexey-pelykh** [comment]
A note to myself: type-alias does work, yet ```[key: string]: TestSubModel2;``` does not

**Comment by alexey-pelykh** [comment]
Hope this will is going to be useful not only to me.

**Comment by alexey-pelykh** [comment]
A related PR - https://github.com/lukeautry/tsoa/pull/77

**Comment by lukeautry** [comment]
Merged #75
